pkg/entrypoint/entrypointer.go

```diff
@@ -114,7 +117,7 @@ func (e Entrypointer) Go() error {
 	}()
 
 	for _, f := range e.WaitFiles {
-		if err := e.Waiter.Wait(f, e.WaitFileContent, e.BreakpointOnFailure); err != nil {
+		if err := e.Waiter.Wait(context.Background(), f, e.WaitFileContent, e.BreakpointOnFailure); err != nil {
```

Comment:
Could we have `Go` take a `context.Context` and use that here? If it's still a `context.Background()` inside `cmd/entrypoint/main.go` that's fine, it's just a bit cleaner and makes it easier to add other stuff like cancelling when we get SIGINT, which we might want to do later.

Reply:
Thank you for the review. Fixed.